build/ninja-rules-cpp.sh

```diff
@@ -226,7 +226,9 @@ setglobal_link_flags() {
       ;;
   esac
 
-  link_flags="$link_flags -Wl,--gc-sections"
+  if test -n "$STRIP_FLAGS"; then
```

Comment:
I think we have to use `${STRIP_FLAGS:-}`, similar to `${OIL_NINJA_VERBOSE:-}`, because we `set -u` in this file

so this will probably fail if neither of the  STRIP_FLAGS is detected

Reply:
Makes sense, updated.